config.yml.erb

```
@@ -520,10 +520,15 @@ db_cluster_id: !Secret
 db_proxy_reporting_endpoint: !Secret
 db_writer_endpoint: !Secret
 
+# Database credentials (username & password) are stored in an AWS Secret in a JSON format required by RDS Proxy:
+# https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-proxy-setup.html#rds-proxy-secrets-arns
+db_credential_admin: !Secret
```

should this be 
```suggestion
db_admin_credential: !Secret
```
for parallelization with the other values here?

---

I went back and forth on this. I’m currently leaning towards the hierarchical naming convention: `db_credential_[user type]`:

- `db_credential_admin`
- `db_credential_writer`
- `db_credential_reader`

I definitely prefer for all the database configuration settings to start with the prefix `db_` but open to suggestions on how to organize the other settings.

Your suggestion has me realizing, I should change the endpoint settings to be consistent:

- `db_endpoint_writer`
- `db_endpoint_reporting`

Or I should change the credentials to the format you suggested.